perf/benchmark_radix_tree.cpp

```diff
@@ -1,6 +1,8 @@
 /* SPDX-License-Identifier: MPL-2.0 */
 
-#if __cplusplus >= 201103L
+#include "../include/zmq.h"
```

Comment:
This is wrong

Reply:
Correct (leftover from some change I undid)